gradle.properties

```diff
@@ -74,10 +74,10 @@ gradle.enterprise.testretry.enabled = false
 systemProp.gradle.enterprise.testretry.enabled = false
 
 # Settings: Languages / Runtimes
-versions.java.language = 22
+versions.java.language = 21
 versions.java.toolchain = 24
-versions.java.minimum = 22
-versions.java.target = 22
+versions.java.minimum = 21
+versions.java.target = 21
```

Comment:
@darvld note for IDEA work

Reply:
We only really need to change the target for `tooling`, `core`, and `base`, but this might be better for the sake of consistency